### ISSUE TITLE
Error "ERR_REQUIRE_ESM" when running sidecar v11.3.18 and 11.3.19

### ISSUE THREAD

**Issue by RomainParracone** [paritytech/substrate-api-sidecar#919]
**Description**

I am running a polkadot node and a substrate api sidecar on CentOS stream 8. 
After upgrading the sidecar to the latest version (v11.3.19), I encountered this error:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /usr/local/lib/node_modules/@substrate/api-sidecar/node_modules/@mangata-finance/types/index.js from /usr/local/lib/node_modules/@substrate/api-sidecar/node_modules/@polkadot/apps-config/cjs/api/spec/mangata.js not supported.
Instead change the require of index.js in /usr/local/lib/node_modules/@substrate/api-sidecar/node_modules/@polkadot/apps-config/cjs/api/spec/mangata.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/usr/local/lib/node_modules/@substrate/api-sidecar/node_modules/@polkadot/apps-config/cjs/api/spec/mangata.js:8:14)
    at Object.<anonymous> (/usr/local/lib/node_modules/@substrate/api-sidecar/node_modules/@polkadot/apps-config/cjs/api/spec/index.js:136:39)
    at Object.<anonymous> (/usr/local/lib/node_modules/@substrate/api-sidecar/node_modules/@polkadot/apps-config/cjs/api/index.js:24:36)
    at Object.<anonymous> (/usr/local/lib/node_modules/@substrate/api-sidecar/build/src/main.js:44:27) {
  code: 'ERR_REQUIRE_ESM'
}
```
I can fix it by editing the file `/usr/local/lib/node_modules/@substrate/api-sidecar/node_modules/@polkadot/apps-config/cjs/api/spec/mangata.js` but I would prefer to avoid manually editing anything.

I have this issue with sidecar v11.3.18 and v11.3.19 but not with older versions. 
I use NodeJS v16.8.0 and also tried using NodeJS v14.17.5 and had the same issue

**Steps to Reproduce**
```
docker run -it quay.io/centos/centos:stream8 bash
dnf -y --nodocs module install nodejs:16
npm install -g @substrate/api-sidecar@11.3.19
substrate-api-sidecar
```

**Comment by TarikGul** [comment]
Similar things have happened in the past with `apps-config` since the dependencies inside of it are for the most part third party chains, that might not have complete compatibility. That being said, I haven't been able to reproduce the above excluding Docker, nor is polkadot/apps seeing this issue. But I will give it more attention tomorrow though as my queue will free up.

**Comment by TarikGul** [comment]
Alright sweet was able to reproduce this! Working on a fix

**Comment by TarikGul** [comment]
Reopening. I will close it once the new release is out and working. The working fix should be reflected in 11.4.1

**Comment by TarikGul** [comment]
@RomainParracone, it should be fixed. Working for me now, can you let me know if its fixed for you with 11.4.1, then we can close the issue.

**Comment by RomainParracone** [comment]
@TarikGul I confirm that v11.4.1 fixed the issue on my side. Thank you!
